aten/src/ATen/templates/TensorBody.h

```diff
@@ -301,6 +301,11 @@ class CAFFE2_API Tensor {
     return !at::impl::variable_excluded_from_dispatch();
   }
 
+  // TODO: do I need to add this method directly on the Tensor to implement conj_view() in UnaryOps?
```

Comment:
I added a getter/setter for conjugate() on the TensorImpl class, but I needed to add the getter to the Tensor class as well in order to flip the state correctly in the conj_view() function, since it deals with Tensors and not TensorImpls. This doesn't feel right though- let me know if I'm missing something.

Reply:
This is OK for a prototype. In the real implementation we'll probably have to bikeshed under what name we should expose this functionality, and probably it should get a native functions schema.